src/components/pages/admin/events/dashboard/hospitality/LastCall.js

```diff
@@ -111,47 +134,100 @@ class LastCall extends Component {
 			});
 	}
 
+	validateFields() {
+		const { lastCallMessage } = this.state;
+		//Don't validate every field if the user has not tried to submit at least once
+		if (!this.submitAttempted) {
+			return true;
+		}
+
+		const errors = {};
+
+		if (lastCallMessage.length > 255) {
+			errors.lastCallMessage = "That message is too long. 255 Char limit";
+		}
+
+		if (lastCallMessage.length < 1) {
+			errors.lastCallMessage = "That message is too short";
+		}
+
+		this.setState({ errors });
+		if (Object.keys(errors).length > 0) {
+			return false;
+		}
+
+		return true;
+	}
+
 	renderConfirmDialog() {
-		const { openConfirmDialog } = this.state;
+		const {
+			openConfirmDialog,
+			isCustom,
+			lastCallMessage,
+			errors,
+			isSending
+		} = this.state;
 		const { classes } = this.props;
 
+		const customLabel = "Send custom call notification";
+		const lastCallLabel = "Send last call notification";
+		const customHeading = "Let your guests know what's going on";
+		const lastCallHeading = "This can only be sent once during your event.";
+		const customDesc =
```

Comment:
For the description, because there is so much that is shared we could combine the two into
```suggestion
		const description = `All attendees who have enabled notifications on their devices will receive the ${isCustom ? 'custom' : 'Last Call'} message`
```